src/components/DataTable/index.jsx

```diff
@@ -74,16 +94,21 @@ class DataTable extends Component {
       headers,
       items,
       title,
+      filters,
     } = this.props;
     const colSpan = (headers && headers.length) || 0;
 
     return (
       <Fragment>
         {title && (
-          <Toolbar>
+          <Toolbar className={classes.toolbar}>
             <Typography variant="title">{title}</Typography>
+            <IconButton onClick={this.handleFilterClick}>
+              <FilterVariantIcon onClick={this.handleFilterClick} />
```

Comment:
Remove the `onClick` from `FilterVariantIcon` since we already have it on `IconButton`.